pontoon/machinery/views.py

```diff
@@ -84,6 +84,15 @@ def concordance_search(request):
     except EmptyPage:
         return JsonResponse({"results": [], "has_next": False})
 
+    # ArrayAgg (used in get_concordance_search_data()) does not support using
+    # distinct=True in combination with ordering, so we need to do one of them
+    # manually - after pagination, to reduce the number of rows processed.
+    projects = Project.objects.order_by("disabled", "-priority").values_list(
+        "name", flat=True
```

Comment:
[performance nitpick] -> the query can filter them by the project names returned in search results?

Reply:
I'm OK with just a single list, I wonder how much work we'd invest into gathering the projects set from all the object_list first.

Reply:
Something like this would work I think:
```python
.filter(name__in=[x for r in data.object_list for x in r["project_names"]])
```

But I don't see any performance wins locally, so I'd rather keep the code less ugly. :)